src/request.go

```diff
@@ -102,7 +102,19 @@ func insertDomainMetrics(eventType string, domain string, beanAttrVals []*beanAt
 	// Create an entity for the domain
 	var e *integration.Entity
 	var err error
-	if args.LocalEntity {
+	if args.RemoteMonitoring {
+		if args.ConnectionURL != "" {
```

Comment:
can we extract this into a function that returns the remoteEntity?

Reply:
will do